trieste/acquisition/rule.py

```diff
@@ -1422,6 +1422,45 @@ def get_initialize_subspaces_mask(
         return tf.logical_not(get_unique_points_mask(centres, tolerance=1e-6))
 
 
+class TREGOBox(TrustRegionBox):
```

Comment:
@vpicheny this is great no? we can now easily have TREGO as a subclass of `TrustRegionBox`